### PR TITLE
Added category navigation, selectors, refactored code

### PR DESCRIPTION
**Pull request by volodymyryaniv**

### CODE REVIEW

File: client/src/components/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        <NavLink
+          to="active"
+          className={({ isActive }) => (isActive ? active : disabled)}
+          onClick={(e) => onClickHandler(e, '/active')}
+        >
+          Active
+        </NavLink>
+        <NavLink
+          to="completed"
+          className={({ isActive }) => (isActive ? active : disabled)}
```
Comment (Kost927) [inline client/src/components/NavBar/NavBar.tsx:43]: duplicated condition 3 times

File: client/src/components/NavBar/NavBar.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          onClick={(e) => onClickHandler(e, '/completed')}
+        >
+          Completed
+        </NavLink>
+      </div>
+      <p className={clearAll} onClick={() => dispatch(clearCompletedTodo())}>
```
Comment (Kost927) [inline client/src/components/NavBar/NavBar.tsx:49]: make a separate function for this, if you'll need to add more logic for onClick there will be a lot of logic in jsx

File: client/src/components/ToDoItem/ToDoItem.tsx [hunk 3]
```diff
@@ -39,19 +37,17 @@ const ToDoItem: FC<ToDoItemTypes> = (props) => {
...
     <article className={container}>
       <div className={content}>
         <div className={mainInfo}>
-          <input
-            className={checkbox}
-            type="checkbox"
+          <ToggleButton
             checked={completed}
-            onChange={() => onSetStatusHandler(id)}
+            onChangeHandler={() => onSetStatusHandler(id)}
```
Comment (Kost927) [inline client/src/components/ToDoItem/ToDoItem.tsx:50]: same, separate func

Comment (volodymyryaniv) [reply client/src/components/ToDoItem/ToDoItem.tsx:50]: Didn`t see it. Thanks

File: client/src/components/NavBar/NavBar.tsx (hunk 2 above)
Comment (volodymyryaniv) [reply client/src/components/NavBar/NavBar.tsx:49]: Ok. No problem

**Review by Kost927 — APPROVED** [review]

**Pull request merged**